lib/inputs/rpm/docker.ts

```diff
@@ -18,6 +18,13 @@ export function getRpmDbFileContent(
       if (typeof stderr === "string" && stderr.indexOf("not found") >= 0) {
         return { stdout: "", stderr: "" };
       }
+
+      if (
+        typeof stderr === "string" &&
+        stderr.toLowerCase().indexOf("no such") >= 0
```

Comment:
how about adding to the string, if we know it's not just ```no such``` but actually ```No such file or directory```?
so longer string and without ```.toLowerCase()```

Reply:
I personally prefer to remove the entire check and never throw :) so I preferred to make it more generic here